Runtime/Behaviours/CinemachineTargetGroup.cs

```diff
@@ -305,17 +302,19 @@ Vector3 CalculateAveragePosition(out float maxWeight)
 
```

Comment:
Changed CalculateAverageOrientation. Tested with a bunch of rotations, and the results seemed more reasonable this way. However, I don't know the use cases of the average rotation on the group. Maybe that could help me determine whether this is a better approach or not.

Also, there is this document for averaging quaternions:
http://wiki.unity3d.com/index.php/Averaging_Quaternions_and_Vectors?_ga=2.247459462.30067401.1582556840-1207821455.1572890629